chat-commands.js

```diff
@@ -1311,7 +1311,7 @@ exports.commands = {
 		Rooms.global.autojoinRooms(user, connection);
 		let targets = target.split(',');
 		let autojoins = [];
-		if (targets.length > 11 || connection.inRooms.size > 1) return;
+		if (targets.length > 11 || connection.inRooms.size > (connection.inRooms.has('lobby') ? 2 : 1)) return;
```

Comment:
On second thought, I think it's enough to move `if (connections.inRooms.size > 1) return;` before `Rooms.global.autojoinRooms(user, connection);`